scout/utils/scout_requests.py

```diff
@@ -113,13 +114,10 @@ def get_request(url):
         LOG.info("Encoded to %s", response.encoding)
     except requests.exceptions.HTTPError as err:
         LOG.warning("Something went wrong, perhaps the api key is not valid?")
-        raise err
```

Comment:
Well found! Hm, the other way of phrasing that is that we didn't handle/try-except the re-raised exception in the second step.  Would not be bad to wrap all calls and flash the user, but it is used in a few places. Perhaps we could import and add a flash directly here?

Reply:
That was also my idea. I'll add the flash here

Reply:
Oh no:

<img width="907" alt="image" src="https://user-images.githubusercontent.com/28093618/208463759-451093b6-9326-4df8-b5c3-9c9d1851c3c8.png">

Since I think there are also some command line functions that are calling this one, I will keep LOG.error, and instead I will flash the error in the server function!

Reply:
Perfect, thanks!

Reply:
I'm not still satisfied, I will make a small fix because I don't think the code as it is works to flash the warning